src/au-table.js

```diff
@@ -101,14 +101,17 @@ export class AureliaTableCustomAttribute {
 
     doFilter(toFilter) {
         let filteredData = [];
-
+        
         for (let next of toFilter) {
             for (let key of this.filterKeys) {
-                let value = next[key].toString();
 
-                if (value.toLowerCase().indexOf(this.filterText.toLowerCase()) > -1) {
-                    filteredData.push(next);
-                    break;
+                if( next[key]){
```

Comment:
If the property in question has a value of 0, it will evaluate to false.

We need to check if next[key] == null || next[key] == undefined